src/services/inlayHints.ts

```diff
@@ -597,6 +598,7 @@ export function provideInlayHints(context: InlayHintsContext): InlayHint[] {
                     Debug.assertNode(node, isParameter);
                     if (node.modifiers) {
                         visitDisplayPartList(node.modifiers, " ");
+                        parts.push({ text: " " });

```

Comment:
I have added this for consistency but I don't know if there is a sane way to test this. Parameter modifiers are only allowed in constructors, right? I think there might not be a way to have them displayed in an inlay hint